packages/components/src/components/pagination/component.tsx

```diff
@@ -300,14 +300,19 @@ export class KolPagination implements API {
 					exportparts="icon"
 					key={`${this.nonce}-${page}`}
 					_customClass={this.state._customClass}
-					_label={`${page}`}
+					_label=""
 					_on={{
 						onClick: (event: Event) => {
 							this.onClick(event, page);
 						},
 					}}
 					_variant={this.state._variant}
-				></kol-button>
+				>
+					<span slot="expert">
+						<span class="visually-hidden">{translate('kol-page')} &nbsp; </span>
```

Comment:
Die Worttrennung eher so?

```tsx
<span slot="expert">
  <span class="visually-hidden">{translate('kol-page')}&nbsp;</span>${page}
</span>
```

Reply:
brauchen wir das Dollarzeichen vor 'page' in diesem Abschnitt? oder meinst du so :
```tsx
<span slot="expert">
  <span class="visually-hidden">{translate('kol-page')}&nbsp;</span>{page}
</span>
```